engine/consensus/dkg/reactor_engine.go

```diff
@@ -181,6 +174,13 @@ func (e *ReactorEngine) startDKGForEpoch(currentEpochCounter uint64, first *flow
 		Interface("members", committee.NodeIDs()).
 		Msg("epoch info")
 
+	_, err = dkgmodule.GetDKGCommitteeIndex(e.me.NodeID(), committee)
+	if err != nil {
+		// node not found in DKG committee bypass starting the DKG
+		log.Warn().Err(err).Msg("node index not found in dkg committee")
```

Comment:
better to be explicit to say something like: node (%v) is not part of DKG committee, skip starting DKG engine

Reply:
In particular, we can log that we were unable to find _ourself_ in the DGK committee. We can add in comments that this indicates the node is un-staking from the network and will not be a participant beginning in the subsequent epoch.

Reply:
https://github.com/onflow/flow-go/pull/1849/commits/aa369fc7718a37db5db1e76cb77481bc0c1cca33